tests/launch_tests.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+SANDBOX="../fssb -- "
+RUNNER="./tests.py"
+
+testcases=(
+	'test_no_syscalls'
+	'test_save_empty_file'
+)
+
+echo "Removing all /tmp/fssb-*"
+rm -rf -- /tmp/fssb-*
+
+echo "Launching tests"
+
+for testcase in ${testcases[@]}
+do
+	$SANDBOX $RUNNER test $testcase
```

Comment:
That is weird. Unless I am missing something, there is a fork in both of the cases:

```bash
➜  tests git:(master) ✗ cat is_forking.sh                               
#!/bin/bash

./tests.py
➜  tests git:(master) ✗ strace -fff -e fork,clone,execve ./is_forking.sh
execve("./is_forking.sh", ["./is_forking.sh"], [/* 77 vars */]) = 0
clone(child_stack=0, flags=CLONE_CHILD_CLEARTID|CLONE_CHILD_SETTID|SIGCHLD, child_tidptr=0x7f82bd8669d0) = 20849
strace: Process 20849 attached
[pid 20849] execve("./tests.py", ["./tests.py"], [/* 76 vars */]) = 0
[pid 20849] execve("/home/dc/bin/python", ["python", "./tests.py"], [/* 76 vars */]) = -1 ENOENT (No such file or directory)
[pid 20849] execve("/home/dc/.local/bin/python", ["python", "./tests.py"], [/* 76 vars */]) = -1 ENOENT (No such file or directory)
[pid 20849] execve("/usr/local/sbin/python", ["python", "./tests.py"], [/* 76 vars */]) = -1 ENOENT (No such file or directory)
[pid 20849] execve("/usr/local/bin/python", ["python", "./tests.py"], [/* 76 vars */]) = -1 ENOENT (No such file or directory)
[pid 20849] execve("/usr/sbin/python", ["python", "./tests.py"], [/* 76 vars */]) = -1 ENOENT (No such file or directory)
[pid 20849] execve("/usr/bin/python", ["python", "./tests.py"], [/* 76 vars */]) = 0
Usage: ./tests.py (test|check) <test_name>
    test    - launches test phase for given test
    check   - launches check&clean phase for given test
[pid 20849] +++ exited with 255 +++
--- SIGCHLD {si_signo=SIGCHLD, si_code=CLD_EXITED, si_pid=20849, si_uid=1000, si_status=255, si_utime=2, si_stime=0} ---
+++ exited with 255 +++
➜  tests git:(master) ✗ vim is_forking.sh 
➜  tests git:(master) ✗ cat is_forking.sh                               
#!/bin/bash

python ./tests.py
➜  tests git:(master) ✗ strace -fff -e fork,clone,execve ./is_forking.sh
execve("./is_forking.sh", ["./is_forking.sh"], [/* 77 vars */]) = 0
clone(strace: Process 20879 attached
child_stack=0, flags=CLONE_CHILD_CLEARTID|CLONE_CHILD_SETTID|SIGCHLD, child_tidptr=0x7fd64d1419d0) = 20879
[pid 20879] execve("/usr/bin/python", ["python", "./tests.py"], [/* 76 vars */]) = 0
Usage: ./tests.py (test|check) <test_name>
    test    - launches test phase for given test
    check   - launches check&clean phase for given test
[pid 20879] +++ exited with 255 +++
--- SIGCHLD {si_signo=SIGCHLD, si_code=CLD_EXITED, si_pid=20879, si_uid=1000, si_status=255, si_utime=0, si_stime=0} ---
+++ exited with 255 +++
```

The only difference should be that `./file.py` would look at the shebang to determine what interpreter to use.

Anyway, I have added `python` into the `RUNNER` variable, so that shouldn't be a problem for you anymore.